### DISCUSSION TITLE
How can we add a route to eth1 interface in security onion 2.9?

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#6347] [Unsupported Versions] [answered]
Hi,

We need to add a route to eth1 interface, but the network interface in not locate in /etc/network/interfaces. I found a old WiKi https://github.com/Security-Onion-Solutions/security-onion/wiki/NetworkConfiguration that was for security onion 14.04/16.04.

Is the same procedure?

Thanks in advance.

**Comment by tomjones1977** [comment, marked as answer]
On CentOS the network config is under `/etc/sysconfig/network-scripts/ifcfg-XXX` where XXX is your interface name.

To add a static route that is different to the default gateway you need to create a file called route-XXX where XXX is your interface name, and put the following line in `ROUTE via IP dev INTERFACE` - for example `192.168.40.0/24 via 192.168.30.1 dev eth0`.

**Comment by ITgeevoSOC** [comment]
Thanks for the info.
